developer-workflow/development-cycle.rst

```diff
@@ -371,3 +371,35 @@ administrators) to merge changes to that branch.
 
 Multi-Factor Authentication must be enabled by the user in order to retain
 access as a Release Manager of the branch.
+
+Governance
+----------
+
+The Python Steering Council has overall authority over Python and has delegated
+some of its responsibilities to other groups.
+
+This table lists the PEPs defining each group's responsibilities,
+and the repository where you can open an issue to ask for a decision.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - PEP
+     - Contact repo
+   * - Steering Council
+     - :pep:`13`
+     - :github:`python/steering-council`
+   * - C API Working Group
+     - :pep:`731`
+     - :github:`capi-workgroup/decisions`
+   * - Documentation Editorial Board
+     - :pep:`732`
+     - :github:`python/editorial-board`
+   * - Typing Council
+     - :pep:`729`
+     - :github:`python/typing-council`
```

Comment:
Just a silly nit to prove I received the notification:

How did you decide to order these? Maybe in PEP order (i.e., roughly chronologically :-) makes sense?

Reply:
I guess "Steering Council" is first because it's primary, and the rest is in roughly alphabetical order. Seems like a reasonable order to me. PEP order doesn't seem that useful as the PEPs were all so close together.

Reply:
Yes, I did consider PEP order but thought SC followed by alphabetical was a bit more useful.